srcpkgs/zed/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'zed'
+pkgname=zed
+version=0.139.3
+revision=1
+archs="x86_64* aarch64*"
+build_style=cargo
+make_install_args="--path crates/zed"
+hostmakedepends="pkg-config protobuf gettext"
+makedepends="alsa-lib-devel fontconfig-devel libxcb-devel libxkbcommon-devel
+libzstd-devel wayland-devel vulkan-loader openssl-devel libgit2-devel"
+depends="git"
+checkdepends="noto-fonts-ttf"
+short_desc="High-performance, multiplayer code editor"
+maintainer="SpidFightFR <spidfight@swisscows.email>"
+license="GPL-3.0-or-later, Apache-2.0, AGPL-3.0-or-later"
+homepage="https://zed.dev/"
+distfiles="https://github.com/zed-industries/zed/archive/v${version}.tar.gz"
+checksum="cd15f80c95ec4c33cbe821ff70e64c3922d1187fa13ac894a8049b615cad2e4f"
+
+post_install() {
+	vlicense ${wrksrc}/LICENSE-AGPL
+	APP_NAME="Zed" DO_STARTUP_NOTIFY="true" APP_ICON="zed" envsubst < ${wrksrc}/crates/zed/resources/zed.desktop.in > ${wrksrc}/crates/zed/resources/zed.desktop
+	sed -i "s/TryExec=zed/TryExec=Zed/" ${wrksrc}/crates/zed/resources/zed.desktop;
```

Comment:
When https://github.com/zed-industries/zed/pull/12951 hits a release you won’t need this anymore.